### PR TITLE
Add a pull request template

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Add a pull request template.
  - What else could we add? CC: @appgurueu

### CODE REVIEW

**Comment by appgurueu** [comment]
We should also link to the Contribution Guidelines alongside the Code of Conduct once the PR is merged.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!--
+  Thank you for contributing to our project!
```
Comment (appgurueu) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: This nicety could be removed to keep the template concise.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  Your PR will be reviewed shortly after submitted.
+
+  Make sure you've read the contributing guidelines and our code of conduct.
```
Comment (appgurueu) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Redundant.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+<!-- Provide a description of the introduced changes in this pull request. -->
+<!-- If the pull request closes an issue, please update the issue ID below. -->
+
+Closes #{ISSUE_ID} <!-- Remove if not applicable -->
+
+### Checklist
+
+- [ ] A description of the changes in this PR is mentioned above.
+- [ ] All the new and existing test pass.
```
Comment (appgurueu) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: Do we have tests for this repo? I'd rather rephrase this as "all checks pass".

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] No plagiarized, duplicated, or repetitive documentation that has been directly copied from another source.
+- [ ] If it's a new explanation, it contains solid, understandable, and accessible information.
+- [ ] I have read the whole [contributing guidelines](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/CONTRIBUTING.md) and agree to the [Code of Conduct](https://github.com/TheAlgorithms/.github/blob/master/CODE_OF_CONDUCT.md).
+
+### Screenshots (if any)
```
Comment (appgurueu) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: I'd probably remove this section - I haven't yet seen an explanation PR that require(d/s) screenshots. If someone really needs it, they can add it themselves.

Comment (Panquesito7) [reply .github/PULL_REQUEST_TEMPLATE.md:23]: But what if the PR is used for something else, such as a new feature?
This PR template is not necessarily only for new explanations.

Comment (appgurueu) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: Alright

**Review by appgurueu — APPROVED** [review]
